src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -2364,9 +2364,9 @@ def test_containerapp_max_inactive_revisions(self, resource_group):
         env = prepare_containerapp_env_for_app_e2e_tests(self)
 
         self.cmd(f'containerapp create -g {resource_group} -n {app} --image {image} --ingress external --target-port 80 --environment {env} --cpu 0.5 --memory 1Gi')
-        self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[JMESPathCheck("properties.configuration.maxInactiveRevisions", None)])
+        self.cmd(f'containerapp show -g {resource_group} -n {app}', checks=[JMESPathCheck("properties.configuration.maxInactiveRevisions", 100)])
```

Comment:
Here is because the default value of maxInactiveRevisions is set to 100 from backend.